lib/api/v1/serializers/result.go

```diff
@@ -2,10 +2,25 @@ package serializers
 
 import (
 	"go-google-scraper-challenge/lib/models"
+	"time"
```

Comment:
```suggestion
	"time"

	"go-google-scraper-challenge/lib/models"
```

Reply:
Fixed in [2e04578](https://github.com/carryall/go-google-scraper-challenge/pull/105/commits/2e0457874316cef64be2692a9aeb1f9a3cce356d)